capstone/install-wazuh.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Fail if any command fails.
+set -e
+
+# Install needed packages.
+apt-get update
+apt-get install curl apt-transport-https lsb-release gnupg2
+
+# Install GPG key.
+curl -s https://packages.wazuh.com/key/GPG-KEY-WAZUH | apt-key add -
+
+# Add the repository.
+echo "deb https://packages.wazuh.com/3.x/apt/ stable main" | tee -a /etc/apt/sources.list.d/wazuh.list
```

Comment:
let's use https://packages-dev.wazuh.com/staging/apt/ to install wazuh 4 instead of wazuh 3.